pkg/testing/fixture.go

```diff
@@ -1196,6 +1196,31 @@ func performConfigure(ctx context.Context, c client.Client, cfg string, timeout
 	return nil
 }
 
+// createTempDir creates a temporary directory that will be
+// removed after the tests passes. If the test fails, the
+// directory is kept for further investigation.
+//
+// If the test is run with -v and fails the temporary directory is logged
+func createTempDir(t *testing.T) string {
+	tempDir, err := os.MkdirTemp("", strings.ReplaceAll(t.Name(), "/", "-"))
+	if err != nil {
+		t.Fatalf("failed to make temp directory: %s", err)
+	}
```

Comment:
Add a log at creation so that it's possible to follow along as the test runs (without needing to wait for the end of the test)
```suggestion
		t.Logf("Created temp directory %q", tempDir)
	}
```

Reply:
I ended up accepting the suggestion, but I've just realised it will duplicate the message on failures.

Do you think it's that necessary to have it printed if someone wants to follow along? Tests usually last for a few seconds, how useful would that be?

Also, if someone wants to follow along, they can just look for the folder that will appear in `/tmp` and follow from there.